client/js/components/procedure/basicSettings/ExportSettings.vue

```diff
@@ -357,29 +370,65 @@ export default {
       this.allChecked = typeof Object.values(this.checkedFields).find(val => val === false) === 'undefined'
     },
 
+    getSingleCheckedField () {
+      const checkedFieldEntries = Object.entries(this.checkedFields).filter(([, value]) => value)
+
+      if (checkedFieldEntries.length === 1) {
+        this.singleCheckedFieldId = checkedFieldEntries[0][0]
+      } else {
+        this.singleCheckedFieldId = ''
+      }
+    },
+
+    handlePreventDefaultForSingleField () {
```

Comment:
As i understand is, this whole code implements the behavior that if only one checkbox is selected, it cannot be unchecked?

I feel that in a perfect world, each checkbox element would only need `:disabled="selectedOptions.length <= 1"` (pseudo code). But as we're using `DpCheckboxGroup` here, things get complicated. I propose to add a `disabled` prop to DpCheckboxGroup, which propagates to all DpCheckbox children.

Reply:
the problem is that by adding the `:disabled` state on a checkbox, it will not be included in the form post action because disabled fields are inactive. Therefore, I need an alternative method to prevent the click action

Reply:
I also considered this approach, but it did not work

Reply:
this is yet another good example of how we might be better off with using a resourceType api2.0 route here, and saving only the section instead of posting the whole form. this would omit the whole problem... idk.